### PR TITLE
SQL digest support inline view

### PR DESCRIPTION
**Pull request by kangkaisen**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] others

## Which issues of this PR fixes ：
for https://github.com/StarRocks/starrocks/issues/4606

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/analysis/InlineViewRef.java [hunk 1]
```diff
@@ -331,4 +331,28 @@ public String tableRefToSql() {
...
+        // Enclose the alias in quotes if Hive cannot parse it without quotes.
+        // This is needed for view compatibility between Impala and Hive.
+        String aliasSql = null;
+        String alias = getExplicitAlias();
+        if (alias != null) {
+            aliasSql = ToSqlUtils.getIdentSql(alias);
+        }
+        if (view != null) {
+            // TODO(zc):
+            // return view_.toSql() + (aliasSql == null ? "" : " " + aliasSql);
```
Comment (Seaven) [inline fe/fe-core/src/main/java/com/starrocks/analysis/InlineViewRef.java:346]: remove

Comment (kangkaisen) [reply fe/fe-core/src/main/java/com/starrocks/analysis/InlineViewRef.java:346]: All codes in this pr don't exist in main branch

**Review by HangyuanLiu — APPROVED** [review]

**Comment by kangkaisen** [comment]
run starrocks_be_unittest

**Comment by kangkaisen** [comment]
run starrocks_clang-format

**Pull request merged**